vm/devices/get/get_resources/src/lib.rs

```diff
@@ -86,6 +86,8 @@ pub mod ged {
         pub secure_boot_template: GuestSecureBootTemplateType,
         /// Enable battery.
         pub enable_battery: bool,
+        /// Suppress attestation and disable TPM state persistence.
```

Comment:
Curious: Is the suppression of attestation for TVM only scenarios? How was it validated to ensure that this path won't regress CVM scenarios?

Reply:
suppressing attestation is the CVM only scenario, but for testing it's possible to use it with TVM.